### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rossteasley**
Edited for smoother English. I'm not familiar with OOP terminology, however, so please check this carefully. Also, since I'm not really familiar with OOP, I'm not a good candidate to edit your in-code comments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,25 +2,25 @@
...
+### Option
 
-Options is the one of component from `DataComponents` series. Options represents the native WordPress options which you can use with `get_option()` or `update_option()` functions from WordPress. But this way of usage is not very handy.
+Option is a component from the `DataComponents` series. Option represents the native WordPress options associated with the `get_option()` or `update_option()` WordPress functions, which are not very convienent.
 
-**The Options component in WP Kit is options on steroids.**
+**The Option component in WP Kit is options on steroids.**
  
- 1. Each time when you are trying to use option you need manually type its name. Without auto complete. And if your product have many options you can get confused. WP Kit solves this issue.
+ 1. Each time you use the WordPress option, you have to manually type its name (unless you're using auto complete), and if your product has many options, you can get confused. WP Kit solves this issue.
```
Comment (tobifjellner) [inline README.md:15]: 1. Each time you use an option in WordPress, you have to...

File: README.md [hunk 2]
```diff
@@ -62,11 +62,11 @@ $option->get();
 
 ### Transients
 
-Transients are very similar to Options. All the difference the only in one thing: `expiration` instead of `autoload`.
+Transients are very similar to Options. The only difference is using `expiration` instead of `autoload`.
 
 ## How to install?
 
-Use Composer. The publishing on WordPress will be in the near future.
+Use Composer. The will be published on the WordPress repository in the near future.
```
Comment (tobifjellner) [inline README.md:69]: It will be published...

File: README.md (hunk 1 above)
Comment (rossteasley) [reply README.md:15]: Good catch.

File: README.md (hunk 2 above)
Comment (rossteasley) [reply README.md:69]: much better.

**Comment by korobochkin** [comment]
Hi. Thank you so much for this improvements. I'm sorry for a long delay which due to WordCamp Moscow and series of things which I had to solve. 😐

**Comment by korobochkin** [comment]
All changes successfully merged :)

**Pull request merged**